src/components/DropdownSelect/DropdownSelect.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import {
+  DivAsInput,
+  Dropdown as DropdownComponent,
+  DropdownTargetProps,
+  Label,
+  H1,
+  Stack,
+  P3,
+  DropdownItemType,
+} from "@deskpro/app-sdk";
+import {
+  faCheck,
+  faExternalLinkAlt,
+  faCaretDown,
+} from "@fortawesome/free-solid-svg-icons";
+import { useMemo } from "react";
+
+interface Status {
+  key: string;
+  value: string;
+  label: JSX.Element;
+  type: string;
+}
+
+type Props = {
+  data?: any[];
+  onChange: (key: string) => void;
+  title: string;
+  value: string;
+  error: boolean;
+  labelName: string;
+  valueName: string;
+  required?: boolean;
+};
+export const DropdownSelect = ({
+  data = [],
+  onChange,
+  title,
+  value,
+  error,
+  labelName,
+  valueName,
+  required,
+}: Props) => {
+  // This works fine but the types are completely wrong
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove unnecessary comments

Reply:
It's explaining why the TS ignore is there